01-path-basics/101-start-here/scripts/lab-ide-build.sh

```diff
@@ -53,6 +53,7 @@ export EKS_SECURITY_GROUPS=$(aws cloudformation describe-stacks --stack-name $AW
 export EKS_SERVICE_ROLE=$(aws cloudformation describe-stacks --stack-name $AWS_MASTER_STACK | jq -r '.Stacks[0].Outputs[]|select(.OutputKey=="EksServiceRoleArn")|.OutputValue')
 
 # Persist lab variables
+echo "AWS_STACK_NAME=$AWS_STACK_NAME" >> ~/.bashrc
```

Comment:
This is a repeat of line 59. My bet is that `master` isn't packaged as part of the release.